docs/markdown/Machine-files.md

```diff
@@ -128,6 +128,14 @@ b = a + 'World'
 a = 'Hello'
 ```
 
+*Since 1.3.0* predefined constant `global_source_root` is the absolute path to
```

Comment:
*the* predefined, *the* project's source, relative to *the*

Reply:
fixed